CMake/FindGSS.cmake

```diff
@@ -97,6 +97,7 @@ if(NOT _GSS_FOUND)  # Not found by pkg-config. Let us take more traditional appr
       # Should also work in an odd case when multiple directories are given
       string(STRIP "${_GSS_CFLAGS}" _GSS_CFLAGS)
       string(REGEX REPLACE " +-I" ";" _GSS_CFLAGS "${_GSS_CFLAGS}")
+      string(REGEX REPLACE " +--isystem" ";" _GSS_CFLAGS "${_GSS_CFLAGS}")
```

Comment:
Did you mean to search for `-isystem`, as opposed to `--isystem`?

In your report the config is returning the former, which is also the
form supported by gcc and llvm/clang.

Reply:
You are right. Sorry about that.